### DISCUSSION TITLE
Help on Getting Started with my existing folder

### DISCUSSION THREAD

**Discussion by xeruf** [beetbox/beets#5329] [Q&A] [answered]
I want to follow https://beets.readthedocs.io/en/stable/guides/main.html#importing-your-library but it is very unclear how it will act on my existing library. I have my library directory set to the directory I will import, and now it says it will copy something around?

Apparently according to the manual I can use `-AC` to prevent it from doing anything to my tracks.
Can I easily do the autotagging later?
I am really cautious because I have over 100GB in my library and no interest in messing it up.

I think it is important to document how a user with an already meeticulously organized librarly would sensibly proceed.

**Comment by amogus07** [comment]
Set your directory to a different one. By default, beets will copy your selected music there (not move it). I am currently in the same situation, and I'm importing my music in batches, you could do the same. After each batch, make sure everything is imported correctly.

**Reply by xeruf** [reply]
That is not what I want. I want to use beets on my whole library immediately, without having it modify anything initially without my consent. I don't have space to duplicate my music library.

**Comment by Serene-Arc** [comment]
I'm a little confused as to what you're asking. If you don't want the files moved or any metadata written, what are you trying to use beets to do? If you use the `move` option, there won't be two copies of the file after it's imported.

**Reply by xeruf** [reply]
I want to use beets on my whole library immediately, without having it modify anything initially without my consent.
For example to run a replaygain and lyrics scan or metadata searches.
I had very particular tagging and folder schemes and want to evaluate which of these I can give up for sane defaults in beets.

**Reply by Serene-Arc** [reply, marked as answer]
Well if you put the following in your config, it shouldn't make any changes to your files, though you should run a test import and make sure. This will construct the beets database but shouldn't change the files at all.

```yaml
import:
  move: no
  copy: no
  write: no
```

**Reply by JOJ0** [reply]
@xeruf does that work for you? IMHO these three options should be what you want, which I understand is "just look around and create a beets database, without changing any files/metadata yet". HTH
